server.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const app = express();
+const path = require("path");
+const fs = require("fs");
+const PORT = process.env.PORT || 5000;
+
+app.use(express.static("public"));
+
+// app.use(express.static("API"))
+
+app.get("/", (request, response) => {
+  response.sendFile(path.resolve("./public/index.html"))
+})
```

Comment:
This works here in this file, but it looks like you aren't actually using your server. You could create another script in your package.json called 'server' or something similar and make the script ```node server.js```. Then in your console you would run ```npm run server``` and then this code would run.